client/src/app/store/effects/app.effects.ts

```diff
@@ -35,6 +35,32 @@ export class Effects {
       })
     )
   );
+  loadMarketClock$: Observable<Action> = createEffect(() =>
+    this.actions$.pipe(
+      ofType(appActions.loadMarketClock),
+      switchMap((action) => {
+        return this.stockService.loadMarketClock().pipe(
+          map((data: any) =>
+            appActions.loadMarketClockSuccess({
+              marketClock: {
+                isOpen: data.body.isOpen,
+                nextClose: data.body.nextClose,
+                nextOpen: data.body.nextOpen,
+                timestamp: data.body.timestamp,
+              },
```

Comment:
we can have a separate function similar to 'formatUser' so the formatting logic is separate